stable/kubernetes-dashboard/templates/svc.yaml

```diff
@@ -13,7 +13,7 @@ spec:
   ports:
   - name: http
     port: 80
-    targetPort: http
+    targetPort: 80
```

Comment:
I basically just dropped in the upstream kubernetes-dashboard stuff into a chart.

I'm not really sure why they did some of the things the way they did.

Are you saying a targetPort can be a reference to a named port in a pod?